src/assert.js

```diff
@@ -0,0 +1,29 @@
+function Assert() {
+
+};
+
+Assert.IsString = function(param) {
+    if (param) {
```

Comment:
Why have you added this check, it is unnecessary. Lets imagine that I have a following function:
```
function printName(name) { ... }
```
And with assert I want to check if I'm providing something
```
function printName(name) {
 Assert.isString(name);
 ...
}
```
When I call it like `printName()`, name will be undefined, and instead of throwing an error, your code will continue to execute. Fix this, otherwise Assert is not doing its job.